modules/gitops-allowing-scm-providers.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * declarative_clusterconfig/managing-app-sets-in-non-control-plane-namespaces.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-allowing-scm-providers_{context}"]
+= Allowing Source Code Manager Providers
+
+[IMPORTANT]
+====
+Please read this section carefully. Misconfiguration could lead to potential security issues. 
+====
+
+Creating role-based access control (RBAC) resources for `applicationSet` resources in non-control plane namespaces can result in the exfiltration of secrets through malicious API endpoints in Source Code Manager (SCM) Provider or Pull Request (PR) generators. To prevent unauthorized access to sensitive information, the Operator disables the SCM Provider and PR generators by default as a precautionary measure.
```

Comment:
```suggestion
Creating role-based access control (RBAC) resources for `ApplicationSet` resources in non-control plane namespaces can result in the exfiltration of secrets through malicious API endpoints in Source Code Manager (SCM) Provider or Pull Request (PR) generators. To prevent unauthorized access to sensitive information, the Operator disables the SCM Provider and PR generators by default as a precautionary measure.
```